client/main.lua

```diff
@@ -12,18 +12,49 @@ HeadCount = 0
 PlayerHealth = nil
 IsDead = false
 IsStatusChecking = false
+
+---@class StatusCheck
```

Comment:
In this particular case, this would be just be a table as there is no prototyping. It's not a big deal but just something to note as it repeats fairly often.

Reply:
Maybe something that would be good for a broader discussion with the team.